tests/js/spec/components/globalSdkUpdateAlert.test.tsx

```diff
@@ -0,0 +1,262 @@
+import moment from 'moment';
```

Comment:
should we name this file `globalSdkUpdateAlert.spec.tsx` ?

Reply:
Yes, 100%